docs/localization.md

```diff
@@ -0,0 +1,143 @@
+You can localize your theme by providing other JSON translation files in the `/messages/` directory. 
+
+[!Note]
+> #### Default language
+> Catalyst hardcodes the `en.json` file as the default language file, which contains U.S. English translations when you set up the project [using the CLI](https://www.catalyst.dev/docs/cli).
+
+## Translation file
+
+To translate your theme, create a JSON file for each language you choose to support. Each language that you want to support must have its own JSON file. These can include non-U.S. versions of English, each with their own spellings.
+
+Name your translation files based on the [BCP 47 specification](https://tools.ietf.org/html/bcp47) of language tags and region codes. For an overview of how these language tags are constructed, see [Language tags in HTML and XML](http://www.w3.org/International/articles/language-tags/).
+
+You can find a list of code subtags in the [IANA Language Subtag Registry](http://www.iana.org/assignments/language-subtag-registry). These subtags are primitives that you can combine to create file name prefixes for individual regions. Here are some examples:
+
+| Localization file name | Corresponding regional language variant | Subtags used |
+| ----------- | ----------- | ----------- |
+| `en.json` | English (default file)| en (English) |
+| `en-US.json` | American English | en (English) + US (United States) |
+| `en-AU.json` | Australian English | en (English) + AU (Australia) |
+| `fr.json` | French | en (French) |
+| `fr-CA.json` | Canadian French | fr (French) + CA (Canada) |
+
+For more examples of frequently-used codes, see [ISO 639-2 Codes for the Representation of Names of Languages](http://www.loc.gov/standards/iso639-2/php/code_list.php).
+
+
+### Translation keys
+
+The JSON files should contain key-value pairs for each translation. You can define translations based on a predefined schema. The translated values you specify will display to shoppers as static string translations.
+
+Use the existing `en.json` file as a template for the schema. The template provides pre-defined keys used to translate the themes in Catalyst's [basic ecommerce functionality](https://www.catalyst.dev/docs#ecommerce-functionality).
+You can only provide translated values for the translation keys specified in the template. 
+
+
+For example, the `en.json` file contains translation keys for 
+
+```json
+"Home": {
+  "Carousel": {
+    "bestSellingProducts": "Best Selling Products",
+    "featuredProducts": "Featured Products",
+    "newestProducts": "Newest products"
+  }
+}
+``` 
+
+In your newly-created JSON file, add a translation of the value in the new language.
+
+```json
+"Home": {
+  "Carousel": {
+    "bestSellingProducts": "Produits les plus vendus",
+    "featuredProducts": "Produits populaires",
+    "newestProducts": "Produits les plus recentsProduits"
+  }
+}
+``` 
+
+
+
+### {} 
```

Comment:
What are the variables in brackets in the `en.json` file? 

For example, where do {rating} and {author} refer to? 
```
"reviewRating": "Rating: {rating} out of 5 stars",
"reviewAuthor": "Posted by {author} on"
```